### PR TITLE
Update pd-control.md

### PR DESCRIPTION
**Pull request by yikeke**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3305/, https://github.com/pingcap/docs-cn/pull/3995
- Other reference link(s):

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -243,16 +243,13 @@ Usage:
...
-> The configuration of the namespace only supports editing `leader-schedule-limit`, `region-schedule-limit`, `replica-schedule-limit` and `max-replicas`.
+    ```bash
+    >> config set hot-region-schedule-limit 4       // 4 tasks of hot Region scheduling at the same time at most
+    ```
 
-```bash
->> config set namespace ts1 leader-schedule-limit 4 // 4 tasks of leader scheduling at the same time at most for the namespace named ts1
->> config set namespace ts2 region-schedule-limit 2 // 2 tasks of region scheduling at the same time at most for the namespace named ts2
-```
+- `hot-region-cache-hits-threshold` is used to set the threshold of a hot Region. A Region is considered as hot only if the number of its cache hits exceed this threshold.
```
Comment (ran-huang) [inline pd-control.md:252]:
```suggestion
- `hot-region-cache-hits-threshold` is used to set the threshold of a hot Region. A Region is considered as hot only if the number of its cache hits exceeds this threshold.
```

File: pd-control.md [hunk 2]
```diff
@@ -476,6 +492,21 @@ Encoding format usage:
...
 
+### `region scan`
+
+Use this command to get all Regions.
+
+Usage:
+
+```bash
+>> region scan
+``
```
Comment (ran-huang) [inline pd-control.md:503]:
```suggestion

```

File: pd-control.md [hunk 3]
```diff
@@ -597,22 +642,27 @@ Usage:
...
->> scheduler add shuffle-leader-scheduler     // Randomly exchange the leader on different stores
->> scheduler add shuffle-region-scheduler     // Randomly scheduling the regions on different stores
->> scheduler remove grant-leader-scheduler-1  // Remove the corresponding scheduler
+>> scheduler show                                 // Display all schedulers
+>> scheduler add grant-leader-scheduler 1         // Schedule all the leaders of the regions on store 1 to store 1
+>> scheduler add evict-leader-scheduler 1         // Move all the region leaders on store 1 out
+>> scheduler add shuffle-leader-scheduler         // Randomly exchange the leader on different stores
+>> scheduler add shuffle-region-scheduler         // Randomly scheduling the regions on different stores
+>> scheduler remove grant-leader-scheduler-1      // Remove the corresponding scheduler
+>> schedule pause balance-region-scheduler 10     // Pause the balance-region scheduler for 10 seconds
```
Comment (rleungx) [inline pd-control.md:658]: should be scheduler?

Comment (yikeke) [reply pd-control.md:658]: See https://github.com/pingcap/docs-cn/pull/3305/files#diff-71fa457167c89bb4e3cb651dc0484326R951. And the latest Chinese doc uses `schedule`. Is it a typo? @rleungx

Comment (yikeke) [reply pd-control.md]: Confirmed it's a typo from the Chinese doc. Fixed in this PR and https://github.com/pingcap/docs-cn/pull/3995.

**Pull request merged**